### ISSUE TITLE
 detect nothing

### ISSUE THREAD

**Issue by wengjinfang** [hizhangp/yolo_tensorflow#80]
WEIGHTS_FILE = None,then train a very big iters,loss still be around 10,and detect nothing,whats problem

**Comment by cenwfohdkue** [comment]
you need use their pre-training file YOLO_small.ckpt, then it works. Otherwise the LOSS will be very big.
#WEIGHTS_FILE = None
WEIGHTS_FILE = os.path.join(DATA_PATH, 'weights', 'YOLO_small.ckpt')

**Comment by JokerKangS** [comment]
> you need use their pre-training file YOLO_small.ckpt, then it works. Otherwise the LOSS will be very big.
> #WEIGHTS_FILE = None
> WEIGHTS_FILE = os.path.join(DATA_PATH, 'weights', 'YOLO_small.ckpt')

if i want to change the net, such as change the cell_size, it won't work when i use yolo_small.ckpt. so, what should i do if i want to train the net from zero?
